queue/data.js

```diff
@@ -171,7 +171,7 @@ Task.prototype.hasClaim = function() {
  *
  * Returns a promise that all expired tasks have been deleted
  */
-Task.expire = async function(now) {
+Task.expire = async function(now = new Date()) {
```

Comment:
Are there places where we call expire() without any date?  I didn't see it, so I think I'm overlooking the need of the default value here.